### DISCUSSION TITLE
Gitpod and Illuminate

### DISCUSSION THREAD

**Discussion by Lupus590** [cc-tweaked/CC-Tweaked#966] [General]
I didn't check before installing it myself, but is Illuminate pre-installed in the Gitpod image? I didn't see it when I quickly went over the Gitpod config stuff in the repo.

It might not be a bad idea to have it there as it does do the Lua lining and Gitpod is recommended for Lua PRs.

**Comment by SquidDev** [comment]
The pre-commit hooks should install and run illuaminate automatically. Are they not being executed?

**Reply by Lupus590** [reply]
I did have a pre-commit hook complaining about python I think. I'll make a note of it when I try to apply your patch on #965

**Reply by Lupus590** [reply]
Not sure about the python thing, but I did notice that I needed to set up a default merge strategy and then I could commit. But illuaminate wasn't working when I tried to run it manually. The patch that you gave me was done in Gitpod so I assume that it's working (the workflow on the PR failing for unrelated reasons I hope - it's a java compile error instead of lua linting).

**Reply by SquidDev** [reply]
Yeah, Java error is some ForgeGradle silliness. I need to pin to the previous version to fix it.

The Python issue might be https://github.com/gitpod-io/gitpod/issues/4886? I was able to fix it by adding `export PIP_USER=no` to my ~/.bashrc and then making a new shell, but it's pretty poor.
